tbx/work/models.py

```diff
@@ -409,8 +409,18 @@ def get_context(self, request, *args, **kwargs):
         except EmptyPage:
             works = paginator.page(paginator.num_pages)
 
-        related_sectors = Sector.objects.all()
-        related_services = Service.objects.all()
+        # Only show Sectors and Services that have been used
```

Comment:
**Question**: These related objects are being calculated based on `self.works`, after `works` has been further filtered in the local context. Is that the desired behaviour (i.e. do not consider the local filtering when deciding what labels to show)?

Ditto regarding blog posts.

Reply:
Yes, we want to show all the labels regardless of the filtering that's been applied